<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiLanguageDialog</class>
 <widget class="QDialog" name="GuiLanguageDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="mp_labelLanguageSite">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mp_labelLanguage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Language</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="mp_leLanguage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="maxLength">
      <number>5</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>159</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="mp_labelPath">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="mp_lePath">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mp_pbSelectFolder">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="statusTip">
        <string>Select language folder</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../beebeep.qrc">
         <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QTreeWidget" name="mp_twLanguage">
     <property name="rootIsDecorated">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="mp_pbDefault">
     <property name="statusTip">
      <string>Restore to default language</string>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>136</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="mp_pbSelect">
     <property name="text">
      <string>Select</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="mp_pbCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_leLanguage</tabstop>
  <tabstop>mp_lePath</tabstop>
  <tabstop>mp_pbSelectFolder</tabstop>
  <tabstop>mp_twLanguage</tabstop>
  <tabstop>mp_pbDefault</tabstop>
  <tabstop>mp_pbSelect</tabstop>
  <tabstop>mp_pbCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../beebeep.qrc"/>
 </resources>
 <connections/>
</ui>
